language/go/config.go

```diff
@@ -41,6 +41,16 @@ import (
 
 var minimumRulesGoVersion = version.Version{0, 29, 0}
 
+type tagSet map[string]bool
```

Comment:
Are we actually using the "present, but mapped to false" state or should this be a `map[string]struct{}`?